packages/arb-token-bridge-ui/src/components/TransferPanel/TokenRow.tsx

```diff
@@ -151,6 +151,12 @@ export function TokenRow({
 
   const tokenBalance = useMemo(() => {
     if (!token) {
+      if (nativeCurrency.isCustom) {
```

Comment:
NIT: To improve readability, I would recommend to create a variable like `isThisTokenErc20NativeCurrency = token === null && nativeCurrency.isCustom` and use it here and on line number 429.

Reply:
These are kind of different cases, so that same variable wouldn't work. The first case needs to first check if it's a token row or not, and then do things differently based on if it's custom fee token network or regular eth.

I did extract a variable to make the JSX a bit more readable: https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1080/commits/13f43f0b5e713f3251e96b277464d6074ab63677